feeds/wifi-trunk/hostapd/files/hostapd.sh

```diff
@@ -810,7 +810,15 @@ hostapd_set_bss_options() {
 				json_get_vars ieee80211w_mgmt_cipher ieee80211w_max_timeout ieee80211w_retry_timeout
 				append bss_conf "ieee80211w=$ieee80211w" "$N"
 				[ "$ieee80211w" -gt "0" ] && {
-					append bss_conf "group_mgmt_cipher=${ieee80211w_mgmt_cipher:-AES-128-CMAC}" "$N"
+					case "$auth_type" in
+						eap192)
+							append bss_conf "group_mgmt_cipher=BIP-GMAC-256" "$N"
```

Comment:
is that not the 256bit cypher as opposed to 192 ?

Reply:
That's what the WPA3 spec calls it:

>  Permitted EAP cipher suites for use with WPA3-Enterprise 192-bit mode are:
> ▪ TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
> - ECDHE and ECDSA using the 384-bit prime modulus curve P-384
> ▪ TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
> - ECDHE using the 384-bit prime modulus curve P-384
> - RSA ≥ 3072-bit modulus
> ▪ TLS_DHE_RSA_WITH_AES_256_GCM_SHA384
> - RSA ≥ 3072-bit modulus
> - DHE ≥ 3072-bit modulus